src/Microsoft.VisualStudio.Editors/OptionPages/OptionPage.vb

```diff
@@ -0,0 +1,74 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.ComponentModel
+Imports System.Windows
+Imports Microsoft.VisualStudio.Shell
+
+Namespace Microsoft.VisualStudio.Editors.OptionPages
+    ' This code is based on corresponding functionality in dotnet\roslyn: https:'github.com/dotnet/roslyn/blob/master/src/VisualStudio/Core/Impl/Options/AbstractOptionPage.cs
+
+    <DesignerCategory("code")>
+    Public MustInherit Class OptionPage
```

Comment:
nit: We should probably ask Roslyn to make their version public and have people inherit from it. Making ours public first means we won't be able to remove it from the API surface later....